### ISSUE TITLE
Make search embeds (document lists) responsive

### ISSUE THREAD

**Issue by reefdog** [documentcloud/documentcloud#312]
Search embeds (document lists) widen and narrow as necessary, but its wrapper height is set statically during init and doesn't overflow, so if narrowing reflows the documents, we lose some docs.

**Comment by knowtheory** [comment]
Yeah, like i mentioned before we'll probably have to rework the markup... my suggestion would be to lose the footer bar and make it a floating header (for forward & back arrows) or something to that effect, and let the list of documents flow below it.

**Comment by reefdog** [comment]
:+1:

Will steal some page embed styling when I do this.
